src/TreeHouse/BuckarooBundle/Request/AbstractTransactionRequest.php

```diff
@@ -37,6 +44,14 @@ public function setInvoiceNumber($invoiceNumber)
     }
 
     /**
+     * @param string|null $description
+     */
+    public function setDescription($description)
+    {
+        $this->description = $description;
```

Comment:
jawel, argument hoeft niet nullable gemaakt te worden zolang het geen typehint heeft, en aangezien kamers nog ver achter loopt met strict types is dat ook niet nodig imo